dist/znk.infra.js

```diff
@@ -2920,11 +2920,14 @@ angular.module('znk.infra.general')
                 this.setter = function(path, newVal){
                     return $q.when(entitySetter(path,newVal));
                 };
+
+                this.entityCache = $cacheFactory('entityCache' + cacheId);
+                cacheId ++;
```

Comment:
In the previous implementation, 2 storages instances would override each other cache for the same path name.
Now each storage instance has its own cache.